src/test/java/com/xml/XmlGenetareTest.java

```diff
@@ -0,0 +1,11 @@
+package com.xml;
+
+import org.junit.Test;
+
+public class XmlGenetareTest {
+    @Test
+    public void testMakeMultiLayerStructure() {
+        XmlGenetare xmlGenetare = new XmlGenetare();
```

Comment:
This test should have assert.

Reply:
I don't know how to write a test correctly
Maybe you will help me?

Reply:
I see Alexander answer in chat. You don't need to use assert.